package.json

```
@@ -6,6 +6,7 @@
     "build:webpack-only": "ts-node --transpile-only ./scripts/next-build-webpack-only.ts",
     "postbuild": "ts-node --transpile-only ./scripts/capture-build-metrics.ts dev-portal",
     "start": "next",
+    "start:withAxe": "AXE_ENABLED=true npm start",
```

super nit: can we use kebab-case here for consistency?
```suggestion
    "start:with-axe": "AXE_ENABLED=true npm start",
```

---

Totally, I didn't even notice this!